Library/Source/NapiBridge.h

```diff
@@ -18,15 +18,15 @@ namespace babylon
             Definition(const Definition&) = delete;
```

Comment:
All changes to this file unnecessary for this code review. Consider reverting? (Or don't bother, since this file is going away anyway.)

Reply:
Yeah, it's going away anyway. Can always revert before going into the main repo if requested.